src/Function/Consequences.agda

```diff
@@ -23,22 +24,22 @@ private
     a b ℓ₁ ℓ₂ : Level
     A B : Set a
     ≈₁ ≈₂ : Rel A ℓ₁
-    f f⁻¹ : A → B
+    to f f⁻¹ : A → B
```

Comment:
I understand that the way variables work, this declaration of `f` and `f⁻¹` will actually have the 'right' things in practice, I find the declaration here confusion, i.e. I would prefer an additional line with `f⁻¹ : B → A` for clarity.

And I thought you'd rename `f` to `to` and `f⁻¹` to `from` everywhere?

Reply:
Re: inlining. Two things about this:
* I'd thought these lemmas weren't used quite as much as they are (I was really focused on their use in `Function.*`), so the case for inlining is less strong now
* that said, the `Function.Consequences.Propositional` version probably *should* be inlined/redefined in that module, because in that setting, the way `cong` and `trans` are defined for `_≡_` means that they take a *much* simpler form than that inheriting from `Setoid`...

Re: `f` and `f⁻¹`: I'd though to have `to` and `from` for the *new* uses (and not simply those in `Structures`/`Bundles`), but I'm not really a fan of 'gratuitous' alpha-conversion elsewhere, if only for the sake of keeping the footprint of the PR smaller. But I could be persuaded... ;-)